src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandlerTest.java

```diff
@@ -179,12 +204,17 @@ public void givenAUcCaseWithSingleElementSelected_thenSetCaseCodeToUs() {
         elementList.add("testElement");
         sscsCaseData.setElementsDisputedList(elementList);
         sscsCaseData.getAppeal().setBenefitType(BenefitType.builder().code("uc").build());
+        
 
         PreSubmitCallbackResponse<SscsCaseData> response = dwpUploadResponseAboutToSubmitHandler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
 
         assertEquals("US", response.getData().getIssueCode());
         assertEquals("001", response.getData().getBenefitCode());
         assertEquals("001US", response.getData().getCaseCode());
+
+        // Ensure that we have cleared the dwpUploadResponseDynamicBenefitType
```

Comment:
Probably don't need these comments all over the place as it's pretty clear what we are trying to do I think

Reply:
Now removed comments